docs/src/modules/ROOT/pages/constraints-and-score/score-calculation.adoc

```diff
@@ -1498,21 +2469,25 @@ public class MyConstraintProviderTest {
 Timefold Solver supports two other types of score calculation.
 
 [#easyJavaScoreCalculation]
-=== Easy Java score calculation
+=== Easy Java/Python score calculation
 
-An easy way to implement your score calculation in Java.
+An easy way to implement your score calculation in Java or Python.
 
 * Advantages:
-** Plain old Java: no learning curve.
+** Plain old Java/Python: no learning curve.
 ** Opportunity to delegate score calculation to an existing code base or legacy system.
 ** Useful for prototyping.
 * Disadvantages:
 ** Slower, typically not suitable for production.
 ** Does not scale because there is no xref:constraints-and-score/performance.adoc#incrementalScoreCalculation[incremental score calculation].
 ** Can not xref:constraints-and-score/understanding-the-score.adoc[explain the score].
 
-To start using Easy Java score calculation, implement the one method of the interface ``EasyScoreCalculator``:
+To start using Easy Java/Python score calculation, implement the one method of the interface ``EasyScoreCalculator``:
```

Comment:
```suggestion
To start using Easy score calculator, implement the one method of the interface ``EasyScoreCalculator``:
```

Reply:
Unsure about that phasing; either "To start using an easy score calculator, ...", or "To start using easy score calculation, ...".